samplers/jaegerremote/example/main.go

```diff
@@ -27,12 +30,19 @@ import (
 )
 
 func main() {
+	// Optional: an implementation of logr.Logger used for demo purposes to catch potential error logs
+	logger := stdr.NewWithOptions(stdlog.New(os.Stderr, "", stdlog.LstdFlags), stdr.Options{LogCaller: stdr.All})
```

Comment:
The addition of this logger is also more than just an update. Wouldn't that be adding quite a bit of noise in the output, making the example less readable?

Reply:
When I wanted to try the example, I couldn't see any error logs from the background updater. By default sampler was using logr.Discard() if I am not wrong, I just added to a simple stdout logr implementation to be able to check error logs while trying. I didn't remove it because I thought that it could be a nice for example.